packages/fxa-event-broker/package.json

```diff
@@ -60,6 +60,7 @@
     "google-auth-library": "^8.0.2",
     "graphql": "^15.6.1",
     "hot-shots": "^9.0.0",
+    "joi": "17.4.0",
```

Comment:
I just noticed that @hapi/joi is still referenced is still a dependcy above. Is there a reason for this?

Reply:
`fxa-event-broker` still has `typesafe-joi` as a dependency, which requires a peer dependency of `hapi/joi@15.x`. When I remove hapi/joi, I find this during `yarn install`:

`➤ YN0002: │ fxa-event-broker@workspace:packages/fxa-event-broker doesn't provide @hapi/joi (p86a06), requested by typesafe-joi`

We have a [ticket](https://mozilla-hub.atlassian.net/browse/FXA-5066) to investigate `typesafe-joi`, but I think we can rewrite it entirely as it has been done in `fxa-shared` in this PR. I'd like to throw it into the next sprint.

Reply:
Okay, sounds good. I saw the changes for typesafe-joi in fxa-shared and didn't now there was more to it. Thanks for filing a follow up!